### ISSUE TITLE
Allow setting `ignore_pipeline_branch_filters` in `bk build new`

### ISSUE THREAD

**Issue by amartani** [buildkite/cli#378]
Creating a build using `bk build new` in a pipeline that has GitHub integration disabled fails with:

```
bk build new --pipeline=$org_name/$pipeline_name
Error: POST https://api.buildkite.com/v2/organizations/$org_name/pipelines/$pipeline_name/builds: 422 Branches have been disabled for this pipeline
```

To create builds for this pipeline via the API, we need to set `ignore_pipeline_branch_filters: true` in the POST call. Could this option be added as a flag for the CLI?

Thank you!

**Comment by mcncl** [comment]
@amartani thanks for raising the issue. I’d suggest using the `api` command for this use case at the moment.

**Comment by raylu** [comment]
this is a rather serious regression that break `bk build` (our main use for the CLI) for most of our pipelines

the pipelines we are most likely to want to trigger manually (via `bk build new`) are the ones most likely to not have the GH integration and be affected by this bug

**Comment by mcncl** [comment]
@raylu @amartani I've opened https://github.com/buildkite/cli/pull/379 to fix this

**Comment by mcncl** [comment]
@amartani @raylu this feature is included in `3.2.0`. Minor bump as there are other additions in there too; supporting `-f` for passing in an environment file, etc
